<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>soundConf</class>
 <widget class="QDialog" name="soundConf" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>332</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>453</width>
    <height>332</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>453</width>
    <height>332</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Configure</string>
  </property>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>9</y>
     <width>431</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title" >
    <string>Sound File:</string>
   </property>
   <widget class="QLineEdit" name="leFile" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>30</y>
      <width>290</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bfSelect" >
    <property name="geometry" >
     <rect>
      <x>330</x>
      <y>30</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text" >
     <string>Select...</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>190</y>
     <width>431</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title" >
    <string>External Program:</string>
   </property>
   <widget class="QLineEdit" name="leProg" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>30</y>
      <width>290</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bpSelect" >
    <property name="geometry" >
     <rect>
      <x>330</x>
      <y>30</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text" >
     <string>Select...</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="bOK" >
   <property name="geometry" >
    <rect>
     <x>220</x>
     <y>280</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bCancel" >
   <property name="geometry" >
    <rect>
     <x>340</x>
     <y>280</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>89</y>
     <width>431</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title" >
    <string>Play With:</string>
   </property>
   <widget class="QRadioButton" name="radioButton3" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>60</y>
      <width>97</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text" >
     <string>ARTS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bpTest" >
    <property name="geometry" >
     <rect>
      <x>330</x>
      <y>60</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text" >
     <string>Test</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton4" >
    <property name="geometry" >
     <rect>
      <x>170</x>
      <y>60</y>
      <width>97</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text" >
     <string>External</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton2" >
    <property name="geometry" >
     <rect>
      <x>170</x>
      <y>20</y>
      <width>98</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text" >
     <string>ESD</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton1" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>98</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text" >
     <string>Internal</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
